h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -533,11 +533,15 @@ protected int nModelsInParallel(int folds) {
   protected int nModelsInParallel(int folds, int defaultParallelization) {
     if (!_parms._parallelize_cross_validation) return 1; //user demands serial building (or we need to honor the time constraints for all CV models equally)
     int parallelization = defaultParallelization;
-    if (_train.byteSize() < 1e6) 
+    if (_train.byteSize() < smalldataSize()) 
       parallelization = folds; //for small data, parallelize over CV models
     return Math.min(parallelization, H2O.ARGS.nthreads);
   }
 
+  protected long smalldataSize() {
```

Comment:
Case on world boundaries, i.e. smallDataSize() ?

Reply:
Yeah, typo - "smalldata" is a one word for me because of our dataset storage ;)